client/src/helpers/test/demo-helper.spec.js

```diff
@@ -0,0 +1,69 @@
+import should from 'should'
+import demoHelper from '../demo-helper'
+import { setAPIUrl } from './testHelper'
+import { prepareAxiosResponse } from './axios-mock-helper'
+import StoreHelper from '../store-helper'
+import mockData from './mockData.json'
+
+jest.mock('axios')
+
+const token = 'demoToken'
+
+describe('demoHelper tests', () => {
+  it('_getApiUrl', done => {
+    setAPIUrl()
+    const storeUrl = StoreHelper.apiUrlGet()
+    const url = demoHelper._getApiUrl()
+    url.should.equal(storeUrl)
+    done()
+  })
```

Comment:
This makes me wonder why the DemoHelper needs it's own method that just does what StoreHelper's method does.  Instead of this test please remove the demo helper method. Let demo-help directly use store helper.  thanks

Reply:
I'll deal with this in another PR